app/src/main/java/com/droidhats/campuscompass/views/SearchFragment.kt

```diff
@@ -24,11 +24,7 @@ import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.droidhats.campuscompass.R
 import com.droidhats.campuscompass.adapters.SearchAdapter
-import com.droidhats.campuscompass.models.Location
-import com.droidhats.campuscompass.models.GooglePlace
-import com.droidhats.campuscompass.models.Building
-import com.droidhats.campuscompass.models.IndoorLocation
-import com.droidhats.campuscompass.models.NavigationRoute
+import com.droidhats.campuscompass.models.*
```

Comment:
Oops these imports got nuked somehow by the IDE in one of my commits. Please fix this :)